subsys/spm/secure_services.c

```diff
@@ -52,6 +52,23 @@ int spm_secure_services_init(void)
 	return err;
 }
 
+#ifdef CONFIG_SPM_SERVICE_READ_FICR
+__TZ_NONSECURE_ENTRY_FUNC
+int spm_request_read_ficr(u16_t offset, u32_t *destination)
+{
+	if (destination == NULL) {
+		return -EINVAL;
+	}
+
+	if ((offset >= 0x204 && offset < 0xC20) ||
```

Comment:
Could we use some defines for all constants here?

Reply:
Yes, we could.